styles/badge/badge.css

```diff
@@ -0,0 +1,66 @@
+.notification {
+    background-color: #555;
+    color: white;
+    text-decoration: none;
+    padding: 15px 26px;
+    position: relative;
+    display: inline-block;
+    border-radius: 2px;
+    margin:1rem;
+
+  }
+  
+  .notification:hover {
+    background: red;
+  }
+
+  
+  .notification .badge {
+    position: absolute;
+    top: -10px;
+    right: -10px;
+    padding: 5px 10px;
+    border-radius: 50%;
+    background: red;
+    color: white;
+  }
+
+  .red-badge {
+    background-color: rgb(182, 3, 3);
+  }
+
+  .badge-down {
```

Comment:
Try being consistent with naming. Like above you used ```red-badge``` then down here used ```badge-down```.  Either prefix or suffix it with badge like in other places you've done already.

Reply:
Sure, Aditya will folloow this rule here after.